projects/igniteui-angular/src/lib/avatar/avatar.component.ts

```diff
@@ -272,20 +271,18 @@ export class IgxAvatarComponent implements OnInit, AfterViewInit {
      *
      * @memberof IgxAvatarComponent
      */
-    get type(): AvatarType {
+    get type(): IgxAvatarType {
```

Comment:
Missing the forth option in the enum will return `undefined` here

Reply:
Added custom to remedy for that.